lib/stringifyRequest.js

```diff
@@ -14,7 +14,7 @@ function isRelativePath(str) {
 
 function stringifyRequest(loaderContext, request) {
 	const splitted = request.split("!");
-	const context = loaderContext.context || (loaderContext.options && loaderContext.options.context);
+	const context = loaderContext.context || loaderContext.rootContext || (loaderContext.options && loaderContext.options.context);
```

Comment:
`rootContext` and `context` are difference, and it is invalid fix, `rootContext` is context for request, `context` is `webpack` context

Reply:
But according to [this](https://webpack.js.org/api/loaders/#this-rootcontext), `this.options.context` has been changed to `this.rootContext`. I thought `this` is just the `loaderContext`. Could you give me some more explanation? Thanks!

Reply:
hm, looks my mistake, can you create new a PR?